docs/index.html

```diff
@@ -54,7 +56,7 @@ <h5>(The rest is a bit more difficult though)</h5>
 $ cd your-module
 $ rspec-puppet-init
                         </code></pre></div>
-                    <p>Then continue on to the <a href="/tutorial/">tutorial!</a></p>
+                    <p>Then continue on to the <a href="{{site.url}}/tutorial">tutorial!</a></p>
```

Comment:
I'd consistently use `baseurl` instead of including the whole domain
```suggestion
                    <p>Then continue on to the <a href="{{site.baseurl}}/tutorial">tutorial!</a></p>
```

Reply:
its better to use site.url across these pages, otherwise it has difficulty loading in the CSS, even when Ive tried to concat the jekyll variables, the CSS still fails to loads. 

This code produces the below output `href="{{site.url}}/{{site.baseurl}}/assets/css/styles.css">`

![image](https://github.com/puppetlabs/rspec-puppet/assets/19972737/5a61001e-1d40-4f59-b96e-799e5554b41c)

From the description of `site.baseurl` you would think its something we need as it mentions using this if your site is served from a sub directory, i find this slightly confusing as we are serving our documentation from `./docs` but we would only need `baseurl` if our project site was `https://puppetlabs.github.io/rspec-puppet/docs` and `./docs` being the subdomain but as thats not the case we do not need `baseurl` as its not served from subdomain